src/string.cr

```diff
@@ -707,11 +707,11 @@ class String
   # Negative indices can be used to start counting from the end of the string.
   #
   # ```
-  # "hello"[0]  # 'h'
-  # "hello"[1]  # 'e'
-  # "hello"[-1] # 'o'
-  # "hello"[-2] # 'l'
-  # "hello"[5]  # raises IndexError
+  # "hello"[0]  # => 'h'
+  # "hello"[1]  # => 'e'
+  # "hello"[-1] # => 'o'
+  # "hello"[-2] # => 'l'
+  # "hello"[5]  # => raises IndexError
```

Comment:
Let's stay consistent with the "raises" in the example below and not use a `=>` here. Also I think `=>` should only be used when that actually returns something.
```suggestion
  # "hello"[5]  # raises IndexError
```